### ISSUE TITLE
bug: binary operation `==` cannot be applied to type `Vec<_>` where `_` was generated by cornucopia

### ISSUE THREAD

**Issue by xoac** [LouisGariepy/cornucopia#48]
After generate code I am getting this error:
```
error[E0369]: binary operation `==` cannot be applied to type `Vec<public::Transfer>`
   --> geyser-plugin/src/cornucopia.rs:289:13
    |
285 |         #[derive(Debug, Clone, PartialEq)]
    |                                --------- in this derive macro expansion
...
289 |             pub transfers: Vec<super::super::types::public::Transfer>,
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this error originates in the derive macro `PartialEq` (in Nightly builds, run with -Z macro-backtrace for more info)
````
generated inside `types` module:
```rust
#[derive(Debug, postgres_types::ToSql, postgres_types::FromSql)]
#[postgres(name = "transfer")]
#[derive(Clone)]
pub struct Transfer {
    pub mint: Vec<u8>,
    pub to_account: Vec<u8>,
    pub from_account: Vec<u8>,
    pub amount: i64,
}
```

generated inside `queries` module:
```rust
#[derive(Debug, Clone, PartialEq)]
pub struct GetTransactionCacheByAnySigner {
    pub signature: Vec<u8>,
    pub fee: i64,
    pub transfers: Vec<super::super::types::public::Transfer>,
    pub signers: Vec<Vec<u8>>,
    pub status: super::super::types::public::TransactionStatus,
    pub slot: i64,
    pub updated_on: time::PrimitiveDateTime,
}
```

The solution to this is to add `PartialEq` to `Transfer`

**Comment by LouisGariepy** [comment]
Hi @xoac, thanks for opening this issue :smile: 

I'm currently working on a complete rearchitecture of Cornucopia, so I'll add this to my list of things to do before 0.8 :+1: 

In general, regarding adding more `derive`s, it's not necessarily trivial to know which trait can be derived for which struct. That is, I have to make sure either the derive works for every single type that cornucopia supports, or create a mechanism to determine which bounds can be added.

Either way, its not a blocker, but I'll have to pour some more thoughts into this. It will be done though soonish though, that I can say.

**Comment by LouisGariepy** [comment]
@xoac This should be fixed in the next release that should come out in a few days :tada:

**Comment by LouisGariepy** [comment]
On track to be solved by #52.
